data/persistence/src/desktopMain/kotlin/social/androiddev/common/persistence/di/DesktopPersistenceModule.kt

```diff
@@ -39,7 +40,13 @@ actual val persistenceModule: Module = module {
     }
 
     single {
-        val driver = JdbcSqliteDriver(url = JdbcSqliteDriver.IN_MEMORY).also { driver ->
+        // refers to the user directory, such us /Users/my_username or /home/my_username or C:\Users\my_username
+        val dbDir = File(System.getProperty("user.home"), "dodo")
```

Comment:
This will make sure Desktop platform stores the data, instead keeping it in memory